equinix/services/fabricv4/docs/AccessPointSelectorType.md

```diff
@@ -2,10 +2,13 @@
 
 Type of Access point
 
-## Properties
+## Enum
 
-Name | Type | Description | Notes
------------- | ------------- | ------------- | -------------
+* `COLO` (value: `'COLO'`)
+
+* `VD` (value: `'VD'`)
+
+* `NETWORK` (value: `'NETWORK'`)
```

Comment:
These kinds of changes are a result of https://github.com/OpenAPITools/openapi-generator/pull/18804, which includes supported enum values in the docs for enum models